### DISCUSSION TITLE
An error occurred during models config fetch error

### DISCUSSION THREAD

**Discussion by marcusprice** [strapi/strapi#8039] [Legacy Topics]
Hello,

I am trying to deploy a strapi on a digitalocean server and have been following this guide: [https://strapi.io/documentation/v3.x/deployment/digitalocean.html](https://strapi.io/documentation/v3.x/deployment/digitalocean.html)

I followed all the instructions in that guide and am coming up with the following error when I try to create the admin account: **"An error occurred during models config fetch error**". What's weird is when I go to mydomain.com/admin (I used a nginx serverblock to direct /admin traffic to localhost:1337) it asks for the username and password, which leads me to believe that the initial admin registration was successfully saved in the database. When I watch the logs in pm2 monit, I don't see any errors appear. 

I've gone through the guide over and over to make sure I didn't miss anything, but I am not getting any different results. Does anyone know what might be causing this issue and what I can do to resolve it?

Info: 

- OS: Ubuntu 20.04 (LTS) x64
- Node Version: 12.18.4
- NPM Version: 6.14.6
- Strapi version: 3.1.4
- Database: postgresql

Edit: when monitoring in the chrome dev tools network tab I see that content-types returns with a 404 response and that the traffic isn't being routed through mydomain.com/admin/content-manager/content-types but instead https://mydomain.com/content-manager/content-types. Could this be an issue with the way I have nginx configured?

**Comment by marcusprice** [comment]
So, I'm pretty sure this problem I'm having is due to my nginx configuration and that what I am looking for is here: https://strapi.io/documentation/v3.x/deployment/nginx-proxy.html#sub-folder-unified-strapi-config

Thought I would post the link in case anyone else is struggling like me. Thanks!

**Reply by derrickmehaffy** [reply]
What do you have configured for your server.js `url` key https://strapi.io/documentation/v3.x/deployment/nginx-proxy.html#sub-folder-unified-strapi-config

**Reply by marcusprice** [reply]
After reading this article I decided I'm going to go the sub folder split route. I saw the bit at the end so I guess it would be this template?

```
module.exports = ({ env }) => ({
  host: env('HOST', '0.0.0.0'),
  port: env.int('PORT', 1337),
  url: 'https://example.com/api',
  admin: {
    url: 'https://example.com/dashboard',
  },
});
```

**Reply by marcusprice** [reply]
My server.js wasn't configured right at all. Realized I was approaching the nginx configuration all wrong. That article was super helpful.

**Reply by yeohwaihong** [reply]
I'm having the same problem here. May I know how did you fix it?

**Reply by derrickmehaffy** [reply]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
